omegaconf/basecontainer.py

```diff
@@ -798,3 +815,27 @@ def _update_types(node: Node, ref_type: type, object_type: Optional[type]) -> No
         if new_ref_type is not Any:
             node._metadata.ref_type = new_ref_type
             node._metadata.optional = new_is_optional
+
+
+def _instantiate_structured_config_impl(
+    retdict: Dict[str, Any], object_type: Type[Any]
```

Comment:
retdict is not a good argument name here.
Also, just pass conf as is. no need to break the object_type out, it's inside.

Reply:
Done in https://github.com/omry/omegaconf/pull/502/commits/072e8d861ce6a8df13fdbe0d67979be3874068cd.